seed-job/template.yaml

```diff
@@ -85,18 +85,14 @@ objects:
                         }
                     }
                     stage('Scan the image') {
-                        sh "docker run --name ${container_name} -d ${image_name} tail -f /dev/null"
+                        sh "docker create --name=scan-data 172.29.33.48:5000/nshaikh/ccp-jenkins-slave"
                         parallel (
                             "list yum update": {
-                                sh (returnStdout: true, script: "docker exec -t ${container_name} yum -q check-update > yum-check-update 2>&1", returnStatus: true)
-                                sh "cat yum-check-update"
-                            },
-                            "rpm -Va":  {
-                                sh (returnStdout: true, script: "docker exec -t ${container_name} rpm -Va > rpm-va 2>&1", returnStatus: true)
-                                sh "cat rpm-va"
+                              sh (returnStdout: true, script: "docker run --rm --volumes-from scan-data --entrypoint /bin/python ${image_name} /opt/scanning/yumupdates.py > yum-check-update 2>&1", returnStatus: true)
+                              sh "cat yum-check-update"
                             }
                         )
-                        sh "docker stop ${container_name} && docker rm ${container_name}"
+                        sh "docker stop scan-data && docker rm scan-data"
```

Comment:
Same as above. We need to use a variable like `container_name` and not hard code the name.